src/python/pants/backend/shell/dependency_inference.py

```diff
@@ -16,7 +16,7 @@
 from pants.core.util_rules.external_tool import DownloadedExternalTool, ExternalToolRequest
```

Comment:
Changes to this file aren't strictly coupled and I can revert them if you'd like. But I noticed it while working on this PR